chalicelib/modules/mongo.py

```diff
@@ -105,14 +105,21 @@ def find_one_document(self, collection: str, query: dict):
             raise
 
     @add_env_suffix
-    def get_all_data_from_collection(self, collection: str):
-        """Fetches all data from the specified collection."""
-        if collection is None:
```

Comment:
Will this case never happen? I'm wondering why this got deleted.

Reply:
Nope it should never occur (it was highlighted as dead code by the interpreter since collection is defined as a `str` type)